Sources/BrowserServicesKit/Common/InfoBundle.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  InfoBundle.swift
+//  DuckDuckGo
+//
+//  Copyright © 2022 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public protocol InfoBundle {
```

Comment:
Does this protocol exist just for testing purposes? (MockBundle was the only usage I could find)

Reply:
I believe it does, and was pulled from iOS?  

I was tackling this myself when working on sync before I moved to that other project.  I'm not keen on it, but the only way to create tests for the Bundle related stuff is to create fake bundles.  I'm think I'd prefer to remove this if possible.

Reply:
InfoBundle is required by AppVersion which I also moved to BSK

Reply:
Okay, if this already existed I'm fine with it